test/ccnl-core/test_prefix.c

```diff
@@ -10,7 +10,7 @@ void test_prefix_to_path()
     char *result = "/path/to/data";
     struct ccnl_prefix_s *p = ccnl_malloc(sizeof(struct ccnl_prefix_s));
     p->compcnt = 3;
-    p->comp = ccnl_malloc(sizeof(char*) * p->compcnt);
+    p->comp = ccnl_malloc(sizeof(unsigned char) * p->compcnt);
```

Comment:
Hmm not quite sure here. Isn't `p->comp` supposed to hold 3 char pointer in this case? The `malloc` below should be `(sizeof(unsigned char))`, though.